src/solvers.jl

```diff
@@ -244,6 +245,7 @@ function solve(m::Model; suppress_warnings=false,
     # limit or tolerance is set (:UserLimit)
     if !(stat == :Infeasible || stat == :Unbounded)
         try
+            objBound = MathProgBase.getobjbound(m.internalModel) + m.obj.aff.constant
```

Comment:
I think this means that we'll silently fail to set the objective value and solution if the solver doesn't implement ``getobjbound``. Make sure you test this on a solver that doesn't implement ``getobjbound`` like Ipopt. Either we need have a special check to see if the solver implements it or you'll have the pleasure of going through every solver package to implement ``getobjbound``.